### PR TITLE
feat: Add files in gh repo and edits to templates for issues and feat…

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**
…ure requests

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+*       @MissingNO57 @pbukva
```
Comment (5A11) [inline .github/CODEOWNERS:1]:
```suggestion
*       @MissingNO57 @pbukva

```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -1,58 +1,82 @@
 name: 🐛 Bug Report
-description: Report a reproducible bug.
-title: "Bug report: <short description>"
+description: Report a bug.
 labels: ["bug", "unconfirmed"]
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/bug-report.yml:3]:
```suggestion
labels: ["bug", "unconfirmed"]
assignees:
  - pbukva
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -1,58 +1,82 @@
...
 body:
   - type: markdown
     attributes:
       value: |
-        Thank you for reporting the issue you are facing. Please complete the following so we can have the details necessary to assist you.
+        Thanks for taking the time to report a bug! We appreciate your contribution to improving the library.
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/bug-report.yml:8]:
```suggestion
        Thanks for taking the time to report a bug! We appreciate your contribution to improving this repository.
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -1,58 +1,82 @@
...
     validations:
       required: true
   - type: textarea
-    id: logs
+    id: environment
     attributes:
-      label: Failure Logs
-      description: Include any relevant log snippets or files here
+      label: Environment Details (Optional)
+      description: Provide any relevant information about your environment, such as operating system, Python version, and any other libraries used.
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/bug-report.yml:63]:
```suggestion
      description: Provide any relevant information about your environment, such as operating system, and any other libraries used.
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/config.yml:1]:
```suggestion
blank_issues_enabled: true
```

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 6]
```diff
@@ -1,49 +1,36 @@
-name: ☝️ Feature request
-description: Suggest an idea for this project
-title: "Feature request: <short description>"
+name: ☝️Feature request
+description: Suggest an idea.
 labels: ["enhancement", "unconfirmed"]
+assignees:
+  - jrriehl
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/feature-request.yml:5]:
```suggestion
assignees:
  - pbukva
```

File: .github/PULL_REQUEST_TEMPLATE/release.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+## Release Details
+
+_[briefly describe the main changes with this release]_
+
+## Checklist
+
+_Put an `x` in the boxes that apply._
+
+- [ ] I am making a pull request against the `master` branch
```
Comment (5A11) [inline .github/PULL_REQUEST_TEMPLATE/release.md:13]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE/release.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] Checks and tests pass locally
+- [ ] I have updated the version in `pyproject.toml`
```
Comment (5A11) [inline .github/PULL_REQUEST_TEMPLATE/release.md:15]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE/release.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] I have regenerated and updated the API docs
```
Comment (5A11) [inline .github/PULL_REQUEST_TEMPLATE/release.md:16]:
```suggestion
```

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] Documentation update.
+- [ ] Something else (e.g., tests, scripts, example, deployment, infrastructure).
+
+## Checklist
+
+_Put an `x` in the boxes that apply:_
+
+- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) guide
+- [ ] Checks and tests pass locally
+
```
Comment (5A11) [inline .github/pull_request_template.md:25]:
```suggestion

<!-- markdown-link-check-disable -->
- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) guide
- [ ] Checks and tests pass locally
<!-- markdown-link-check-enable-->

```

File: CITATION.cff
Comment (5A11) [inline CITATION.cff:1]: delete this file

File: docs/CONTRIBUTING.md
Comment (5A11) [inline docs/CONTRIBUTING.md:1]: This file should be integrated with the `DEVELOPING.md` in the other PR. 

Please revert the changes and remove and integrate in the PR that adds `DEVELOPING.md`.

File: .github/PULL_REQUEST_TEMPLATE/release.md
Comment (5A11) [inline .github/PULL_REQUEST_TEMPLATE/release.md:1]: Delete this file

File: docs/CONTRIBUTING.md [hunk 11]
```diff
@@ -1,31 +0,0 @@
-# Documentation Setup
```
Comment (5A11) [inline docs/CONTRIBUTING.md:1]: Integrate with DEVELOPING.md. 

Revert this delete. In the PR for developing.md, delete this and ask (in the pr description) that its content get integrated.

File: docs/CONTRIBUTING.md
Comment (5A11) [inline docs/CONTRIBUTING.md:1]: Integrate with DEVELOPING.md.

Revert this delete. In the PR for developing.md, delete this and ask (in the pr description) that its content get integrated.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 12]
```diff
@@ -1,58 +1,84 @@
...
+        - label: I checked the [existing issues](https://github.com/fetchai/fetchd/issues) and made sure there are no similar bug reports
           required: true
+  - type: dropdown
+    id: category
+    attributes:
+      label: Category
+      description: Select the category that best describes the bug.
+      options:
+        - Bug (unexpected behavior)
+        - Documentation Issue (error or unclear information in docs)
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/bug-report.yml:28]:
```suggestion
```

File: .github/CODEOWNERS [hunk 13]
```diff
@@ -0,0 +1 @@
+*       @MissingNO57 @pbukva @ Josumner
```
Comment (5A11) [inline .github/CODEOWNERS:1]:
```suggestion
*       @MissingNO57 @pbukva @Josumner
```

File: .github/CODEOWNERS [hunk 14]
```diff
@@ -0,0 +1 @@
+*       @MissingNO57 @pbukva @Josumner
```
Comment (5A11) [inline .github/CODEOWNERS:1]:
```suggestion
*       @MissingNO57 @pbukva @Jonathansumner
```

File: docs/CONTRIBUTING.md
Comment (5A11) [inline docs/CONTRIBUTING.md:1]: This file should be integrated with the DEVELOPING.md in the other PR.

Please revert the changes here. Then remove the file and integrate with DEVELOPING.md in the PR for it.

**Review by MissingNO57 — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 15]
```diff
@@ -1,58 +1,83 @@
...
+    validations:
+        required: false
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional Information (Optional)
+      description: Include any screenshots, code snippets, or other relevant details that might help us understand the bug.
+      render: markdown
+    validations:
+      required: false
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/bug-report.yml:83]:
```suggestion
      required: false

```
